messages/zh-CN/app.php

```diff
@@ -4,622 +4,624 @@
+    'This information will be public. Integration for "log in with twitter" can be done from the profile page.' => '这些信息会被公开。可以从用户页面绑定Twitter帐号。',
+    'This Month (UTC)' => '本月（UTC）',
+    'This option is provided for deleting an incorrectly-reported battle.' => '该选项用于删除错误上传的对战。',
+    'This twitter account has already been integrated with another user.' => '已经有用户绑定该Twitter帐号。',
+    'This website doesn\'t support Internet Explorer. Please use a modern browser, for example Chrome or Firefox.' => '本网站不支持IE。请使用现代网络浏览器，比如Chrome或Firefox。',
+    'This website has color-blind support. Please check "Color-Blind Support" in the "Username/Guest" menu of the navbar to enable it.' => '本网站提供色盲辅助。请在导航栏的“用户/游客”菜单下勾选“色盲辅助”。',
+    'This website is an open source project. It is under the MIT License. The source code is available on GitHub.' => '本网站是开源项目。基于MIT授权。源代码可在GitHub查看。',
+    'This website is an UNOFFICIAL SERVICE. It is not related to the Splatoon development team or Nintendo.' => '本网站不是官方服务。与任天堂的Splatoon团队没有关系。',
+    'This {attribute} is already in use.' => '该{attribute}已经启用。',
+    'Time' => '时间',
+    'Time is up' => '时间到',
+    'Times' => '时间',
+    'Time was up' => '时间到',
+    'Time Zone' => '时区',
+    'Title' => '称号',
+    'Title (After)' => '称号（之后）',
+    'Today' => '今天',
+    'Total' => '总计',
+    'Total Clout' => '影响力总计',
+    'Total Clout (After the battle)' => '影响力总计（对战之后）',
+    'Total Inked' => '涂地总计',
+    'Trends' => '趋势',
+    'Turf Inked' => '涂地面积',
+    'Turf inked (including bonus)' => '涂地面积（包含奖励）',
+    'Turf Inked + Bonus' => '涂地面积和奖励',
+    'TV' => '电视',
+    'Twitter @name' => 'Twitter @昵称',
+    'TZ' => '时区',
+    'Unknown' => '未知',
+    'Unlink account' => '取消绑定',
+    'Update' => '更新',
+    'Update Your Email Address' => '更新你的电子邮件地址',
+    'Update Your Icon' => '更新头像',
+    'Update Your Password' => '更新密码',
+    'Update Your Profile' => '更新档案',
+    'Upload icon' => '上传头像',
+    'Upload new image' => '上传新图片',
+    'URL related to this battle' => '这场对战的网址',
+    'Use %' => '使用 %',
+    'Use % and Win %' => '使用 % 和 赢 %',
+    'Used in global stats' => '用于全局统计',
+    'Use full width of the screen' => '使用全部的屏幕宽度',
+    'Use profile icon of your twitter account' => '使用Twitter登录头像',
+    'User Agents in last 24 hours' => '最近一天的客户端',
+    'User Count' => '用户总数',
+    'User Name' => '用户名字',
+    'Users' => '用户',
+    'Use your profile icon' => '使用档案头像',
+    'Verification Code' => '验证码',
+    'Verify' => '验证',
+    'Version' => '版本',
+    'Version {0}' => '版本 {0}',
+    'View Settings' => '显示设置',
+    'We\'ll send an email when you log in to the website or change your password.' => '登录网站或是更改密码后我们会给你发送一封邮件。',
     'We\'ll send an email when you log in to the website or change your password.' => '',
```

Comment:
Could you remove this line?

Reply:
Yes, I must accidentally added it when resolving merge conflicts.